ci/pipelines/deploy-to-test.yml

```diff
@@ -587,7 +589,7 @@ jobs:
         trigger: true
       - get: pay-infra
       - get: pay-ci
-      - load_var: toolbox_image_tag
+      - load_var: tag
```

Comment:
Is there a reason you've gone back to calling this `tag` rather than say `app_image_tag`?

Reply:
Yep originally the deploy for Toolbox was an inline tag. Since I'm using the `deploy_params` and `deploy-app.yml` has `TAG` I had to change it to that. A future PR will rename `tag` to `app_image_tag`.